chcemvediet/apps/anonymization/management/commands/attachment_anonymization.py

```diff
@@ -0,0 +1,65 @@
+from optparse import make_option
+
+import magic
+from django.core.files.base import ContentFile
+from django.core.management.base import BaseCommand, CommandError
+
+from poleno.attachments.models import Attachment
+from chcemvediet.apps.anonymization.models import AttachmentFinalization
+from chcemvediet.apps.inforequests.models import Action
+
+
+class Command(BaseCommand):
+    help = u'Creates AttachmentFinalization for the specified Attachment.'
+
+    option_list = BaseCommand.option_list + (
+        make_option(u'-f', u'--file',
+                    dest=u'filename',
+                    help=u'define file path'),
+        make_option(u'--content_type',
+                    help=u'define content type of file',
+                    ),
+        make_option(u'--debug',
+                    default=u'',
+                    help=u'add debug message'
+                    ),
+        make_option(u'--force',
+                    action=u'store_true',
+                    help=u'overwrite an existing successful AttachmentFinalization'
+                    ),
+    )
+
+    def usage(self, subcommand=None):
+        return u'manage.py attachment_anonymization attachment [options] [file]'
```

Comment:
Zoznam argumentov by mal byt zadefinovany tak, ze defaultna `usage` metoda vygeneruje to co treba. Ak vygeneruje nieco inac, tak mame zle definovane argumenty commandu.

Si si nevsimol, ze sa ti ani nevypisuje obsah `help`?

Ak chces vypisat pozicne argumenty, pozri napriklad `datacheck/management/commands/datacheck.py` ako.